crates/node/src/networking/p2p/noise.rs

```diff
@@ -173,7 +173,10 @@ impl Encoder<Bytes> for Codec {
     fn encode(&mut self, msg: Bytes, dst: &mut BytesMut) -> Result<(), Self::Error> {
         match self.noise {
             State::Handshake(ref mut noise) => {
-                let msg_len = noise.0.write_message(&msg, &mut self.buffer).unwrap();
+                let msg_len = noise
+                    .0
+                    .write_message(&msg, &mut self.buffer)
+                    .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?;
```

Comment:
Same here. This "diff" is already in `main`: https://github.com/gevulotnetwork/gevulot/blob/main/crates/node/src/networking/p2p/noise.rs#L176-L179 :thinking:

Reply:
The same.

Reply:
I've rebased from master. Some commit  came back. I think it's because individual commit are squash so rebase can't detect them.
I think it's ok, I've done some local test.